sklearn/datasets/tests/test_openml.py

```diff
@@ -1158,3 +1158,16 @@ def test_fetch_openml_raises_illegal_argument():
 
     assert_raise_message(ValueError, "Neither name nor data_id are provided. "
                          "Please provide name or data_id.", fetch_openml)
+
+
+@pytest.mark.parametrize('gzip_response', [True, False])
+def test_fetch_openml_with_ignored_feature(monkeypatch, gzip_response):
+    # 62 is the ID of the ZOO dataset
```

Comment:
The test is good. Could you add a first comment as:

```
# Regression test for #14340
```

Reply:
I added the line.